docker-compose.yml

```diff
@@ -90,9 +90,6 @@ services:
       - DATA_DIR=./data
       - RSTUF_BROKER_SERVER="amqp://guest:guest@rabbitmq:5672"
       - RSTUF_REDIS_SERVER="redis://redis"
-      - RSTUF_AUTH=true
-      - SECRETS_RSTUF_TOKEN_KEY="secret"
-      - SECRETS_RSTUF_ADMIN_PASSWORD="secret"
```

Comment:
Please, let's make such changes in a specific commit with a commit message with arguments as to why we do that, so we can reference this later.

Reply:
That's for future prs.

Reply:
My point was to do that in another commit, not another pr.